edge/pkg/eventbus/mqtt/client.go

```diff
@@ -83,7 +83,7 @@ func onSubConnect(client MQTT.Client) {
 		token := client.Subscribe(t, 1, OnSubMessageReceived)
 		if rs, err := util.CheckClientToken(token); !rs {
 			klog.Errorf("edge-hub-cli subscribe topic: %s, %v", t, err)
-			return
```

Comment:
If Subscribe failed here. do we need to panic directly?

Reply:
Well，I use an empty topic to test, actually It's just  an invalid topic not a panic, I think the invalid topics such as an empty topic should not affect other valid topics. And  I want to know in which case subscribe will  failed due to panic because I haven't seen it during my study.

Reply:
The `SubTopics` will be used in kubeedge, so we need to ensure that every top must be subscribed successfully.